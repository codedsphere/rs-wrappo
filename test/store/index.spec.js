import configStore from 'store';
import rootReducer from '../__mocks__/rootReducer';
import rootSaga from '../__mocks__/rootSaga';

describe('store', () => {
  it('should have a store', () => {
    const {store, persistor} = configStore({
      rootReducer,
      rootSaga,      
      middlewareAdd: [],
    });
    expect(store).toMatchSnapshot();
    expect(persistor).toMatchSnapshot();
  });
});
