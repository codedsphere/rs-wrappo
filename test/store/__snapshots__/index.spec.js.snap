// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`store should have a store 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(Symbol.observable): [Function],
}
`;

exports[`store should have a store 2`] = `
Object {
  "dispatch": [Function],
  "flush": [Function],
  "getState": [Function],
  "pause": [Function],
  "persist": [Function],
  "purge": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(Symbol.observable): [Function],
}
`;
